string picker_id
int32 n_trays
string curr_node
---

